spec/features/teachers/user_can_login_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "rails_helper"
+
+describe "As a teacher" do
+  before :each do
+    @teacher = create(:teacher)
+    visit welcome_path
+  end
+
+  it "I can log in successfully" do
+    create(:course,2)
+    create(:student,4)
+    @teacher.courses << Course.first
+    Student.first.courses << Course.first
+    Student.second.courses << Course.first
+    Student.third.course << Course.second
+    Student.fourth.course << Course.second
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@teacher)
+
+    expect(page).to eq("Welcome to HereToLearn")
```

Comment:
Does this pass? Even though there is more on the page than just "Welcome to HereToLearn?"

Reply:
typo - fixed!